.github/workflows/release.yaml

```diff
@@ -37,4 +41,7 @@ jobs:
           git config --global user.name "Mykhailo Kuznietsov"
           git config --global user.email "mkuznets@redhat.com"
           export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-          /bin/bash make-release.sh --version ${{ github.event.inputs.version }} --trigger-release
+          if [[ ${{ github.event.inputs.noCommit}} == "true" ]]; then
+            no_commit="--no-commit"
```

Comment:
why would we want to bump the version but not commit it? if it's the same value, surely the commit would be a no-op?